sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1971,8 +1971,8 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
           s"""
              |CREATE TABLE t(a int, b int, c int, d int)
              |USING parquet
-             |PARTITIONED BY(a, b)
              |LOCATION "${dir.toURI}"
+             |PARTITIONED BY(a, b)
```

Comment:
Is it a relevant change? Since the PR is about ORDER-INSENSITIVENESS, can we keep the original code instead of making an irrelevant change like this?

Reply:
This is an end-to-end test for `ORDER-INSENSITIVENESS`. I do not want to introduce a new one for it

Reply:
Oh, I see. +1.